CMakeLists.txt

```diff
@@ -1,53 +1,51 @@
-# http://ros.org/doc/groovy/api/catkin/html/user_guide/supposed.html
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.5)
 project(depthimage_to_laserscan)
+if(NOT WIN32)
+  set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++14 -Wall -Wextra -Wpedantic")
+endif()
 
-# Load catkin and all dependencies required for this package
-find_package(catkin REQUIRED dynamic_reconfigure image_geometry image_transport nodelet roscpp sensor_msgs)
-#find_package(OpenCV REQUIRED)
+find_package(ament_cmake REQUIRED)
 
-# Dynamic reconfigure support
-generate_dynamic_reconfigure_options(cfg/Depth.cfg)
+find_package(image_geometry REQUIRED)
+find_package(OpenCV REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(sensor_msgs REQUIRED)
 
-catkin_package(
-  INCLUDE_DIRS include
-  LIBRARIES DepthImageToLaserScan DepthImageToLaserScanROS DepthImageToLaserScanNodelet
-  CATKIN_DEPENDS dynamic_reconfigure image_geometry image_transport nodelet roscpp sensor_msgs
+include_directories(include
+  ${OpenCV_INCLUDE_DIRS}
 )
 
-include_directories(include ${catkin_INCLUDE_DIRS})
+add_library(DepthImageToLaserScan
```

Comment:
Oh, I see.  I can't make it shared right now, because when I added the image_geometry library, I did not make that one SHARED.  I'll go and change that one as well.